### DISCUSSION TITLE
Help needed Manx agent  shell taking commands not returning any output

### DISCUSSION THREAD

**Discussion by Baguma-r** [mitre/caldera#2230] [Q&A] [answered]
Hi 
Kindly help
Agent connecting but no output
![screenshot](https://user-images.githubusercontent.com/89108174/129809108-0c6f5e6b-00f4-4a63-8c5d-2f3a87d14299.JPG)
 
 combed internet but don't seem to find a solution 
~$  ls
~$  ./scanner.sh #{target.ip}
~$  cd
~$  
~$  
~$  if [ -f ~/.ssh/authorized_keys ]; then ssh_authorized_keys=$(cat ~/.ssh/authorized_keys); echo $ssh_authorized_keys > ~/.ssh/authorized_keys; fi;
~$  
~$  
~$  
~$  
~$

**Comment by alexanderkent** [comment, marked as answer]
Are there any errors in the browser console? Press Command+Option+J (Mac) or Control+Shift+J (Windows, Linux) to jump into the Console panel of Chrome DevTools.

**Reply by Baguma-r** [reply]
Thanks for the guidance . I managed to see the error in the  browser console . I am using AWS EC2 instance for the setup, the security group was blocking port 7012
